developers/index.md

```diff
@@ -6,15 +6,21 @@ title: Developer Guide
 
 # Overview
 
-
-The _Developer Guide_ contains practical and technical information for people willing to contribute to the development of openHAB or of one of its add-ons.
-
-This manual is an on-going work.
-
-.
-
-.
-
-.
-
-.
+The _Developer Guide_ contains practical and technical information for people that want to get into the development of openHAB.
+
+- Get an overview of how the openHAB project work is [organised](contributing/governance).
```

Comment:
Forgot a `.html` here, too late! (for this PR) 😞

Reply:
I can add that in my test pr for single lines in lists.
So it is only a short temporary problem. If you would have a look at that one and merge it.